decorator.rb

```diff
@@ -0,0 +1,28 @@
+require_relative 'nameable'
+
```

Comment:
- [ ] As the different reviewers mentioned , you forgot to add `attr_accessor :nameable`  :cry:  . Please note that this is very important because it will make the nameable attribute accessible by instance :+1:  Please fix it in order to make app works fine :+1: 

```
class Decorator < Nameable

     attr_accessor :nameable

      def initialize(nameable)
        super()
        @nameable = nameable
     end

// The rest of your code //
```

Reply:
I don't understand.. there's an attr_accessor :nameable just before the method